### ISSUE TITLE
Scrollbar backgrounds are different colours

### ISSUE THREAD

**Issue by Cassolotl** [tootsuite/mastodon#6154]
This is on ephemeral.glitch.social, which is on 2.1.0.

![screen_shot_2018-01-02_at_09_41_01](https://user-images.githubusercontent.com/6607512/34480007-c44f7542-efa1-11e7-96b1-b9f0d2bfc9a0.png)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by ykzts** [comment]
cc @akihikodaki

**Comment by akihikodaki** [comment]
tl;dr: That is intentional. The dark scrollbar is preferred in List column, and Getting Started column has similar UI. Therefore Getting Started column has the dark scrollbar for consistency.

The scrollbar is darker because the background of link collections is darker than the timeline. Though there is no margin in Getting Started column and the background is not visible, there is margin in List column and you can see the background. If you don't make the color darker, the scrollbar cannot be distinguished from the background.
![localhost_3000_web_lists 1](https://user-images.githubusercontent.com/17036990/34506777-2d461b7e-f073-11e7-9841-90fb0aa7bc19.png)

**Comment by akihikodaki** [comment]
https://github.com/tootsuite/mastodon/issues/6154#event-1407190896: I closed this by mistake.
